bench_route.go

```diff
@@ -0,0 +1,496 @@
+package goyave
```

Comment:
This file should be included inside the test file to avoid being compiled and using memory in applications using the framework.

Reply:
oh, i will move it to file bench_test.go